app/main.py

```
@@ -1,6 +1,11 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": 18,
+        "column": 80,
+        "message": "line too long (99 > 79 charactres)",
+        "name": "E501",
+        "souce": "flake8"
+    }
```

This function should use the `error` parameter to dynamically create the returned dictionary. Currently, it returns hardcoded values. For example, the value for the `"line"` key should come from `error["line_number"]`.

Also, on line 7, the key `"souce"` is misspelled; it should be `"source"` as per the requirements.